app/src/geckoBeta/java/org/mozilla/fenix/engine/GeckoProvider.kt

```diff
@@ -66,7 +66,7 @@ object GeckoProvider {
         val geckoRuntime = GeckoRuntime.create(context, runtimeSettings)
         val loginStorageDelegate = GeckoLoginStorageDelegate(
             storage,
-            { context.settings().shouldAutofillLogins && context.settings().shouldPromptToSaveLogins }
+            { context.settings().shouldPromptToSaveLogins }
```

Comment:
Can you comment here, that as a quickfix for #8967 we're conflating the two settings?